sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -205,16 +205,17 @@ def process_encoded(self, encoded_windowed_values):
       self.output(decoded_value)
 
   def try_split(self, fraction_of_remainder, total_buffer_size):
+    # type: (...) -> Optional[Tuple[int, Optional[Tuple[operations.DoOperation, common.SplitResultType]], Optional[Tuple[operations.DoOperation, common.SplitResultType]], int]]
```

Comment:
Some named tuples, or even a named type, could help for `Optional[Tuple[operations.DoOperation, common.SplitResultType]]`